units/cloud_cleanup.go

```diff
@@ -67,21 +67,44 @@ func (j *cloudCleanupJob) Run(ctx context.Context) {
 		j.env = evergreen.GetEnvironment()
 	}
 
-	cloudManager, err := cloud.GetManager(ctx, j.env, cloud.ManagerOpts{
-		Provider: j.Provider,
-		Region:   j.Region,
+	accountRoles := j.env.Settings().Providers.AWS.AccountRoles
+	/*
```

Comment:
I think this is too much context, the comment on L77 is sufficient, you can remove this comment block.